index.js

```diff
@@ -6,76 +6,13 @@ function ask(questionText) {
     rl.question(questionText, resolve);
   });
 }
-function getRandomIntInclusive(min, max) {
-  min = Math.ceil(min);
-  max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-}
-
-async function secondGuess() {
-  let tryAgain = await ask("Higher or Lower?  ");
-  if (tryAgain.toLowerCase() === "lower") {
-    await ask("How about " + getRandomIntInclusive(min, max) + "? ");
-  } else if (tryAgain.toLowerCase() === "higher") {
-    await ask("How about " + getRandomIntInclusive(min, max) + "? ");
-  } else if (tryAgain.toLowerCase() === "correct") {
-    console.log("I'm so smart!!!");
-  }
-
-};
-async function guessNum() {
-  let firstGuess = await ask("Are you thinking of the number..." + computerGuess + "? \nYes or No? ");
-  if (firstGuess.toLowerCase() === "correct") {
```

Comment:
You're specifically asking for yes or no in your question, it would be more user friendly for the correct response to be 'yes'